PowerShellLibrary/Add-AdAppRegistrations.ps1

```diff
@@ -506,10 +508,62 @@ Function Add-AdAppRegistrations() {
     }
 }
 
+Function Add-FederatedCredential() {
+    [CmdletBinding(SupportsShouldProcess)]
+    Param(
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string]$appRegJsonPath,
+        [Parameter(Mandatory = $false)]
+        [string]$graphApiversion = "v1.0"
+    )
+
+    $apps = Get-Content -Raw -Path $appRegJsonPath | ConvertFrom-Json
+    
+    foreach ($app in $apps.applications) {
+        $appReg = Get-AzADApplication -DisplayName $app.displayName       
+
+        $federatedCredentials = Get-AzADAppFederatedCredential -ApplicationObjectId $appReg.id
+        $federatedCredentials | Select-Object -Property Name
+
+        $ficName =  $app.displayName + "-fic"
+        $issuer = "https://vstoken.dev.azure.com/" + $app.adoOrganizationId
+        $subject = "sc://defragovuk/DEFRA-FFC/" + $app.subscriptionName + "-FSC"
```

Comment:
Can "defragovuk/DEFRA-FFC" this be added dynamically?

Reply:
Also remove -FSC

Reply:
updated